src/app/images.tsx

```diff
@@ -28,20 +31,23 @@ export default function Images({
             });
         } else {
             selected = await open({
-                multiple: true,
-                filters: [
-                    {
-                        name: "Image",
-                        extensions: ["jpg", "jpeg", "JPG", "JPEG", "CR2"],
-                    },
-                ],
+                multiple: true
             });
         }
         if (Array.isArray(selected)) {
-            assets = selected.map((item: any) => convertFileSrc(item));
+            set_image_error(false)
+            for (let i = 0; i < selected.length; i++) {
+                if (Extensions(selected[i]) != "jpg" && Extensions(selected[i]) != "jpeg" && Extensions(selected[i]) != "JPG" && Extensions(selected[i]) != "JPEG" && Extensions(selected[i]) != "CR2") {
```

Comment:
I think we're supporting TIFF too, which I think would correspond to the extensions `.TIFF`, `.tiff`, `.TIF`, and `.tif`